messages/variables/messages.go

```diff
@@ -2,25 +2,34 @@ package variables
 
 var (
 	// [ variables ]
-  
-	Usage 				= "variables"
-	ShortDescription   = "Manage your variables on the Azion Edge platform"
-	LongDescription    = "Manage your variables' varaibles on the Azion Edge platform"
-	FlagHelp           = "Displays more information about the Rules Engine command"
-	FlagId              = "Unique identifier of the Variable"
+	Usage            = "variables"
+	ShortDescription = "Create variables for edges on Azion's platform"
+	LongDescription  = "Build your Web applications in minutes without the need to manage infrastructure or security"
```

Comment:
Just copy + paste the same description as the one you used for Short Description. @gabriel-azion will later on edit these messages